open-craft/opencraft
# BB-2426: Add email verification page

This PR replaces the Ocim email confirmation page by a page in the new frontend.

**Testing instructions:**
1. Checkout this branch.
2. Add these variables to your `.env`:
```
USER_CONSOLE_FRONTEND_URL="http://localhost:3000/"
NEW_USER_CONSOLE_REGISTRATION_ENABLED=True
```
3. Make sure you have email set up on your devstack:
```
EMAIL_BACKEND='django.core.mail.backends.smtp.EmailBackend'
# SMTP configuration
EMAIL_HOST=smtp.gmail.com
EMAIL_USE_TLS=True
EMAIL_PORT=587
EMAIL_HOST_USER=YOUR-ACCOUNT@opencraft.com
EMAIL_HOST_PASSWORD=<PASSWORD>
``` 
4. Register a new instance.
5. Click the link in your email, check that you were taken to the new frontend, and wait until the confirmation page appears. 
6. Check that the instance starts provisioning.
7. Try a random link: http://localhost:3000/verify-email/1234/ and check that the email verification fails.

**Reviewer:**
